backend/src/main/java/com/exadel/carinsurance/service/implementation/AuthService.java

```diff
@@ -75,9 +80,21 @@ public ResponseEntity signup( @RequestBody RegisterRequestEntity request ) {
   }
 
   @Override
-  public ResponseEntity login( @RequestBody AuthRequestEntity request ) {
+  public ResponseEntity login( @RequestBody AuthRequestEntity request,
+                               HttpServletRequest httpServletRequest,
+                               HttpServletResponse httpServletResponse ) {
     String userEmail = request.getEmail();
 
+    Cookie[] cookies = httpServletRequest.getCookies();
+    if ( cookies != null ) {
+      for ( Cookie cookie : cookies ) {
+        cookie.setMaxAge( 0 );
+        cookie.setPath( "/" );
+
+        httpServletResponse.addCookie( cookie );
+      }
+    }

```

Comment:
Why do you need this? What is the intention of copy paste all cookies from client with overriden properties?

Reply:
You can check this cheat sheet for more details on cookie lifecycle https://crypto.stanford.edu/cs142/lectures/cookie.html

Reply:
Removed this part of code